docs/walkthrough.md

```diff
@@ -0,0 +1,87 @@
+# Walkthrough
+
+In the following, we gonna show you how to use `kubicorn` to ramp up a Kubernetes cluster in AWS, use it and tear it down again.
+
+As a prerequisite, you need to have `kubicorn` installed. Since we don't have binary releases yet, we assume you've got Go installed and simply do:
+
+```
+$ go get github.com/kris-nova/kubicorn
+```
+
+The first thing you will do now is to define the cluster resources. For this, you need to select a certain profile. Of course, once you're more familiar with `kubicorn`, you can go ahead and extend existing profiles or create new ones.
+In the following we'll be using an existing profile called `aws`, which is—surprise, surprise—a profile for a cluster in AWS.
+
+Now execute the following command:
+
+```
+$ kubicorn create --name myfirstk8s --profile aws
+```
```

Comment:
Perhaps add a comment that this doesn't Do Things on the cloud yet, it just generates local files with the intentions which later command really apply.  (I figured it out, but not until scrolling down and noticing "apply" and "oh I didn't give it credentials yet, riiight")